cpp/rss_aggregator.cc

```diff
@@ -274,15 +280,19 @@ constexpr unsigned SECONDS_TO_MILLISECONDS = 1000;
 
 int ProcessFeeds(const std::string &subsystem_type, const std::string &xml_output_filename, DbConnection * const db_connection,
                  Downloader * const downloader) {
+    unsigned number_feeds_with_error = 0;
     db_connection->queryOrDie("SELECT * FROM tuefind_rss_feeds WHERE FIND_IN_SET('" + subsystem_type + "', subsystem_types) > 0");
     auto result_set(db_connection->getLastResultSet());
     while (const auto row = result_set.getNextRow()) {
         LOG_INFO("Processing feed \"" + row["feed_name"] + "\".");
-        const unsigned new_item_count(ProcessFeed(row["id"], row["feed_name"], row["feed_url"], row.getValue("title_suppression_regex"),
+        const int new_item_count(ProcessFeed(row["id"], row["feed_name"], row["feed_url"], row.getValue("title_suppression_regex"),
                                                   row.getValue("descriptions_and_substitutions"), row.getValue("strptime_format"),
                                                   downloader, db_connection,
                                                   StringUtil::ToUnsigned(row["downloader_time_limit"]) * SECONDS_TO_MILLISECONDS));
-        LOG_INFO("Downloaded " + std::to_string(new_item_count) + " new items.");
+        if (new_item_count == ERROR_DOWNLOAD or new_item_count == ERROR_PARSING)
```

Comment:
If there is no difference in handling the specific errors, we could also compare it against < 0 (this would be more robust in case we add more error types later).